Aside from the android exception, this also fails Travis CI:
```
error[E0405]: cannot find trait `ToCss` in this scope
     --> /home/travis/build/servo/servo/target/geckolib/debug/build/style-87781de7ccb3d936/out/properties.rs:31332:10
      |
31332 |     impl ToCss for FeatureTagValue {
      |          ^^^^^ not found in this scope
      |
      = help: possible candidates are found in other modules, you can import them into scope:
                `use cssparser::ToCss;`
                `use style_traits::ToCss;`
                `use style_traits::values::ToCss;`
error[E0412]: cannot find type `FeatureTagValue` in this scope
     --> /home/travis/build/servo/servo/target/geckolib/debug/build/style-87781de7ccb3d936/out/properties.rs:31332:20
      |
31332 |     impl ToCss for FeatureTagValue {
      |                    ^^^^^^^^^^^^^^^ not found in this scope
error[E0433]: failed to resolve. Use of undeclared type or module `fmt`
     --> /home/travis/build/servo/servo/target/geckolib/debug/build/style-87781de7ccb3d936/out/properties.rs:31333:67
      |
31333 |         fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
      |                                                                   ^^^^^^^^^^ Use of undeclared type or module `fmt`
error[E0433]: failed to resolve. Use of undeclared type or module `fmt`
     --> /home/travis/build/servo/servo/target/geckolib/debug/build/style-87781de7ccb3d936/out/properties.rs:31333:46
      |
31333 |         fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
      |                                              ^^^^^^^^^^^ Use of undeclared type or module `fmt`
error: cannot continue compilation due to previous error
error: Could not compile `style`.
```

--- a/components/style/values/generics/mod.rs
+++ b/components/style/values/generics/mod.rs
@@ -181,12 +181,12 @@ impl<T> OneOrMoreCommaSeparated for FontSettingTag<T> {}
 
 impl<T: ToCss> ToCss for FontSettingTag<T> {
     fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
-        use byteorder::{WriteBytesExt, BigEndian};
+        use byteorder::{BigEndian, ByteOrder};
         use cssparser::serialize_string;
         use std::str;
 
-        let mut raw: Vec<u8> = vec!();
-        raw.write_u32::<BigEndian>(self.tag).unwrap();
+        let mut raw = [0u8; 4];
+        BigEndian::write_u32(&mut raw, self.tag);
         serialize_string(str::from_utf8(&raw).unwrap_or_default(), dest)?;
 
         self.value.to_css(dest)
@@ -310,5 +310,3 @@ impl ToCss for FontSettingTagFloat {
         self.0.to_css(dest)
     }
 }
-
-

